src/marqo/tensor_search/backend.py

```diff
@@ -62,7 +63,9 @@ def get_index_info(config: Config, index_name: str) -> IndexInfo:
 
 def add_customer_field_properties(config: Config, index_name: str,
                                   customer_field_names: Iterable[Tuple[str, enums.OpenSearchDataType]],
-                                  multimodal_combination_fields: Dict[str, Iterable[Tuple[str, enums.OpenSearchDataType]]]):
+                                  multimodal_combination_fields: Dict[str, Iterable[Tuple[str, enums.OpenSearchDataType]]],
+                                  max_retry_attempts: int = DEFAULT_MARQO_MAX_BACKEND_RETRY_ATTEMPTS,
```

Comment:
Since the codepath for this is  `add_customer_field_properties` -> `put` -> `send_request`, wouldn't it be better to set the default `max_retry_attempts` and `max_retry_backoff_seconds` here to `None`, so that `send_request` catches it and fills it in with the constant?